aries-site/src/tests/navigation/deeplinks.js

```diff
@@ -0,0 +1,52 @@
+
+/* eslint-disable no-undef */
+import { ReactSelector, waitForReact } from 'testcafe-react-selectors';
+import { Selector } from 'testcafe';
+import { baseUrl } from '../utils';
+
+// A Deeplink is a subsection of the sections. This is testing if the page is taken to the right section of page to match the hash
+
+// Selector name found using Chrome React dev tools on prod mode of website
+const Search = 'D Box';
+
+const getSuggestion = page => {
+  return ReactSelector(`${Search} StyledDrop Button`).withText(page);
+};
+
+fixture('Search')
```

Comment:
I think my thought is that this file would have one fixture called "Deep linking" and that the two tests would fall beneath it. What do you think?

Reply:
I see your point I agree I will change that